### PR TITLE
Unit Testing Updates

### PR DESCRIPTION
**Pull request by jacob6838**
- [x] Fixing UUID unit tests (broken by re-factoring NAMESPACE_UUID to environment variable)
- [x] Fixing unit test warnings
    - [x] Creating schema_retrieval.py to pull in external schema references, to resolve jsonschema external schema reference warnings
    - [x] Adding documentation to README describing resolution to warnings from package versions
    - [x] Ignoring urllib3.exceptions.SystemTimeWarning (tests use timemachine library to mock datetime, to match timestamps between messages). This was configured in pytest.ini

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,6 +179,15 @@ python -m pytest 'tests/' -v
 
 Ensure you have your environment configured correctly (as described above).
 
+### Unit Test Warnings
+
+There are a few warnings shown by pypi, based on old package versions. These can be resolved by running the following commands:
```
Comment (payneBrandon) [inline README.md]: Is there a corresponding requirements.txt we should update for this?

Comment (jacob6838) [reply README.md]: Good point, updated! The protobuf library is a dependency, so doesn't need to be specified directly in the requirements.txt

Comment (jacob6838) [reply README.md]: Updated the README as well

**Comment by jacob6838** [comment]
I created a task to resolve the unit test warnings, when newer packages are released:
https://dev.azure.com/SOC-OIT/CDOT/_backlogs/backlog/CDOT%20Connected%20Vehicles/Backlog%20items/?workitem=233094

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**